### ISSUE TITLE
Play "neutral" music & skip track improvements

### ISSUE THREAD

**Issue by gammaw** [mmmnmnm/lahmacun#234]
What is "between"? Less than an hour?

**Comment by gammaw** [comment]
After various experiments (with LS 1.4.2: https://github.com/mmmnmnm/lahmacun/issues/237) I couldn't find a solution that would extend our current logic naturally. As an alternate I've come up with the following workaround, which would change our robot DJ logic significantly but using the same established LS structure (which we know that works). The following proposal also contains a workaround for the fade out bug (https://github.com/mmmnmnm/lahmacun/issues/240). 

The brave new world would look like this: 
- The "default" playlist would be an ambient playlist (now off air is our default playlist). This new ambient playlist would play between shows, e.g., between Kezek and Erdenklang (approx. 30 minutes). 
- We would explicitly schedule off air blocks, e.g., from midnight until the start of the first show of the day, but we could schedule them during the day too, when we have long breaks.
- A workaround for the fade out could be that there would be no returning tracks. Every track starts from the beginning. This could be achieved via the `source.skip` operator (see https://github.com/savonet/liquidsoap/issues/530). 

@pvj @baxgas what do you think? 

Risks of the approach:
- After show jingles shouldn't play at the end of off air blocks. Hopefully it won't be the case when they last until the scheduled show starts.  
- Hopefully source.skip will work.

**Comment by gammaw** [comment]
`Source.skip` works nicely. -> Off air songs are skipped when a show starts and a new will start after the show ends. -> There is no need of fading in. 

An unsolved issue is the following: since `playlist_jingle_azura` is played alternating with `playlist_off_air`, `playlist_jingle_after_show` will always be followed by a song from `playlist_jingle_azura`. Unfortunately, double skipping doesn't do it because skip skips within the playlist (which is `playlist_off_air` in this case).

**Comment by gammaw** [comment]
`Source.skip` issue asked upstream: https://github.com/savonet/liquidsoap/issues/1217

**Comment by gammaw** [comment]
@pvj @baxgas FYI: ambient blocks have been added following the logic we discussed. Tests are looking good on dev, hopefully it can go live these days. 

**Note**: every block ends 30 minutes into the last show of the block, e.g., on Friday the ambient block ends at 21:30 "during" Arcade-ok. This is because we allow time in case it's a live show. As a result, if the show never takes place, or the delay > 30 minutes, then there will be an ambient -> off air transition with a donate jingle (which is not nice given that some donate jingles refer to the end of a show, e.g. "Did you like the show..." but such cases should be rare).
